src/pages/home/ReportScreen.tsx

```diff
@@ -102,7 +102,7 @@ type ReportScreenProps = OnyxHOCProps & CurrentReportIDContextValue & ReportScre
 function getReportID(route: ReportScreenNavigationProps['route']): string {
     // The report ID is used in an onyx key. If it's an empty string, onyx will return
     // a collection instead of an individual report.
-    return String(route.params?.reportID || '');
+    return String(route.params?.reportID || 0);
```

Comment:
@ishpaul777 Why is this change required? Shouldn't the previous code return the same output as well?